README.md

```diff
@@ -6,6 +6,23 @@ a viewer component that displays JSON schemas (built using [material-ui](https:/
 - `react` 16.8+
 - `react-dom` 16.8+
 
+## Usage
+Install package
+```
+npm i material-ui-json-schema-viewer
+```
+Import package and use component
+```js
+import SchemaViewer from 'material-ui-json-schema-viewer';
+
+<SchemaViewer schema={jsonSchema} references={schemaReferences} />
+```
+
+| Prop | Type | Required | Description |
+|:---|:---|:---|:---|
+| `schema` | Object | ✓ | A JSON schema object. |
+| `references` | Array| ✓ | An array of JSON schema objects (including the schema to render), which the schema may refer to. (each schema within the references must include an `$id` property to use for key-value mapping the references) |
```

Comment:
Is `references` required even if the schema does not include a ref?

Reply:
`references` is not required when there is no ref. Fixed this part!